client/src/main/java/com/paypal/selion/internal/platform/grid/browsercapabilities/ChromeCapabilitiesBuilder.java

```diff
@@ -58,4 +67,12 @@ private String getBinaryPath() {
                 Config.getConfigProperty(ConfigProperty.SELENIUM_CHROMEDRIVER_PATH));
         return location;
     }
+
+    /*
+     * Returns the comma separated chromedriver chromeoptions as a List of Strings after white space removal.
+     */
+    private List<String> getChromeOptions() {
+        String chromeOptions = Config.getConfigProperty(ConfigProperty.SELENIUM_CHROME_OPTIONS);
+        return Arrays.asList(chromeOptions.split("\\s*,\\s*"));
```

Comment:
As coded, this will never get a string with `commas` in it. The call to `getconfigProperty()` will always return just the first item.